.github/tools/run_mdadm_tests.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/bash
+
+sudo make clean
+sudo make -j$(nproc)
+sudo make install
+sudo mdadm -Ss
+sudo ./test setup
+sudo ./test --skip-broken --no-error --disable-integrity --disable-multipath --disable-linear --keep-going
```

Comment:
we should provide test status up:
```suggestion
ret=$(sudo ./test --skip-broken --no-error --disable-integrity --disable-multipath --disable-linear --keep-going)
sudo ./test cleanup
exit(ret)
```